pytorch_lightning/trainer/training_io.py

```diff
@@ -124,6 +124,12 @@
     list, tuple, set, dict,
     Namespace,  # for back compatibility
 )
+try:
+    from omegaconf import DictConfig, OmegaConf
+except ImportError:
+    pass
+else:
+    PRIMITIVE_TYPES = PRIMITIVE_TYPES + (DictConfig, OmegaConf)
```

Comment:
1. DictConfig is not a primitive type.
2. OmegaConf is not really a type, it's more of an API.

Reply:
this is not used in the end...